src/components/assetlib_projects/asset_dialog.tscn

```diff
@@ -0,0 +1,52 @@
+[gd_scene load_steps=4 format=3 uid="uid://birwt63v5rr5c"]
+
+[ext_resource type="Script" path="res://src/components/assetlib_projects/asset_dialog.gd" id="1_7i63p"]
+[ext_resource type="PackedScene" uid="uid://bcn7ypx0qusqp" path="res://src/components/projects/import_project_dialog/import_project_dialog.tscn" id="2_2yi4x"]
+[ext_resource type="PackedScene" uid="uid://ct6r5vx7pxmuu" path="res://src/components/assetlib_projects/assetlib_project_saving_dialog.tscn" id="3_6stsb"]
+
+[node name="AssetDialog" type="ConfirmationDialog"]
+title = ""
+position = Vector2i(0, 36)
+size = Vector2i(466, 333)
```

Comment:
In general, every size property (including custom_minimum_size) should be scaled with Config.EDSCALE. So, every such a property has to be set from script like so: self.hbox_container.size = Vector2(466, 333) * Config.EDSCALE  

I usually handle it with custom_minimum_size and recommend doing it in the same way